app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { ListItemType } from '@/app/app-types'
+import { faker } from '@faker-js/faker'
+
+describe('List-item component: test cases', () => {
+  const items: ListItemType[] = [
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
+  ]
+  test('render list-item component', () => {
+    render(<ListItem items={items} />)
+    const listItemEl = screen.getByTestId('list-item-wrapper')
+    expect(listItemEl).toBeInTheDocument()
+  })
+
+  test('load the list elements', () => {
+    const listItems: ListItemType[] = [
```

Comment:
You can declare data at top of file once and use it again instead of redeclaring it.